### ISSUE TITLE
Esta versão incorpora inúmeras pequenas correções de bugs contribuídas por meio de pull requests para o repositório GitHub. Obrigado a todos os colaboradores

### ISSUE THREAD

**Issue by Anapaulacrf-code** [apprenticeharper/DeDRM_tools#2333]

**Comment by Anapaulacrf-code** [comment]
Não consigo usar e nem mesmo configurar, basta instalar e qdo vou configurar ja fecha o calibre, já tentei de tudo, (Dentro do que sei , claro) pois não sou "entendida"do assunto, uso calibre há tempos outros plugins também começaram a dar alguns BOS, acredite se quiser, troquei meu computador, uma evolução enorme apenas pelo calibre, para que aproveite mais o que ele oferece em versões mais recentes, eu uso esse software demais, absurdamente, mas to apanhando e quero-me qualquer maneira usar esse plugin, até mesmo chat GBT eu "deixei louco", caso consiga me ajudar, mesmo sabendo que não sou entendida no assinto de códigos e tudo ais eu agradeço, deixo aqui mu email. anapaula@bookbupe.com

**Comment by ElleKayEm** [comment]
Update to DeDRM 10.0.9: https://github.com/noDRM/DeDRM_tools/releases/tag/v10.0.9

**Comment by Anapaulacrf-code** [comment]
ja, tentei novamente, agora e naooooo vai. acredito que eu nao estou sabendo configurar, ou tirar do kindle via cabo, de algum jeito, é so abrir já esta td no calibre, mas nao, abre nao converte, nada, tentei, 3 versões, já, ou pode ser algo do computador talvez, dias atras tive problemas com ebook translator, também, quando eu usava um pc velho que mal fazia atualizações, nada acontecia, agora só quebro a cabeça.

**Comment by ElleKayEm** [comment]
How are you downloading the Kindle books?

**Comment by Anapaulacrf-code** [comment]
Vou te responder em português e vou usar um tradutor, o que for melhor pra vc me avise ok? 
Não estou baixando, apenas com o cabo e o calibre aberto eles já estão no calibre. Baixei alguns que também não funcionou. Através do site da Amazon, também copiei e colei os arquivos que eu queria pra outro local. 
Eu nunca nem tinha usado o Kindle via cabo. Portanto eu tive um problema na Amazon com defeitos em 3 dispositivos recentes e hoje não quiseram trocar novamente. Por isso eu fiquei na tentativa de não ser politicamente correta. Entende? Quero arrancar tudo do KU porque passei o maior nervoso. E outros motivos também. Tenho uma biblioteca extensa que compartilho não publicamente e sou
Moderadora do zlibrary. Faço trabalhos demais compartilhando educação e conhecimento gratuitamente mas da maneira certa. Eu moro no Brasil. E não entendo absolutamente nada, do que diz aí, desculpe toda explicação, mas brasileiro sabe como é neah…. Fala até ! Eu quero usar o plugin e vou continuar tentando. Comprei até um computador melhor pra isso. Uso o plugin de traduzir e já forma uns 6000 mil arquivos. Só de tradução. Entao…. Quero usar este plugin e tirar os ebooks de lá que são quase 200 e mais os do KU também quero. 😯
Ana Paula Carvalho

> Em 17 de jul. de 2024, à(s) 00:12, ElleKayEm ***@***.***> escreveu:
> 
> How are you downloading the Kindle books?

**Comment by Anapaulacrf-code** [comment]
I will answer you in Portuguese and I will use a translator, whatever is best for you, let me know, ok?
I'm not lowering it, just with the handle and the caliber open they are already in the caliber. I downloaded some that didn't work either. Through the Amazon website, I also copied and pasted the files I wanted to another location.
I had never even used the Kindle via cable. So I had a problem with Amazon with defects in 3 recent devices and today they didn't want to exchange them again. That's why I tried not to be politically correct. He understands? I want to tear everything away from KU because I was so nervous. And other reasons too. I have an extensive library that I share non-publicly and I am
zlibrary moderator. I do a lot of work sharing education and knowledge for free but in the right way. I live in Brazil. And I don't understand absolutely anything about what you say there, sorry for all the explanation, but Brazilians know what it's like neah…. Speak up! I want to use the plugin and will keep trying. I even bought a better computer for this. I use the translate plugin and it already creates around 6000 files. Just translation. Then…. I want to use this plugin and get the ebooks out of there, which are almost 200 and the ones from KU I also want
Ana Paula Carvalho

> Em 17 de jul. de 2024, à(s) 00:25, ***@***.*** escreveu:
> 
> Vou te responder em português e vou usar um tradutor, o que for melhor pra vc me avise ok?
> Não estou baixando, apenas com o cabo e o calibre aberto eles já estão no calibre. Baixei alguns que também não funcionou. Através do site da Amazon, também copiei e colei os arquivos que eu queria pra outro local.
> Eu nunca nem tinha usado o Kindle via cabo. Portanto eu tive um problema na Amazon com defeitos em 3 dispositivos recentes e hoje não quiseram trocar novamente. Por isso eu fiquei na tentativa de não ser politicamente correta. Entende? Quero arrancar tudo do KU porque passei o maior nervoso. E outros motivos também. Tenho uma biblioteca extensa que compartilho não publicamente e sou
> Moderadora do zlibrary. Faço trabalhos demais compartilhando educação e conhecimento gratuitamente mas da maneira certa. Eu moro no Brasil. E não entendo absolutamente nada, do que diz aí, desculpe toda explicação, mas brasileiro sabe como é neah…. Fala até ! Eu quero usar o plugin e vou continuar tentando. Comprei até um computador melhor pra isso. Uso o plugin de traduzir e já forma uns 6000 mil arquivos. Só de tradução. Entao…. Quero usar este plugin e tirar os ebooks de lá que são quase 200 e mais os do KU também quero

**Comment by ElleKayEm** [comment]
Sorry, I don't understand.  DRM is removed by DeDRM when you add a book to calibre.  For books from or for an E-ink Kindle, you need that Kindle's serial number entered in DeDRM's customization.  For books downloaded with Kindle for PC, calibre needs to be on the same computer.  You also need the KFX Input plugin: https://www.mobileread.com/forums/showthread.php?t=291290

**Comment by Anapaulacrf-code** [comment]
I'll try again, just like you say, 😉 tks

**Comment by Anapaulacrf-code** [comment]
can you help me? I will be grateful. What's the setup like here?Ana Paula CarvalhoEm 17 de jul. de 2024, à(s) 09:30, ElleKayEm ***@***.***> escreveu:﻿
Sorry, I don't understand.  DRM is removed by DeDRM when you add a book to calibre.  For books from or for an E-ink Kindle, you need that Kindle's serial number entered in DeDRM's customization.  For books downloaded with Kindle for PC, calibre needs to be on the same computer.  You also need the KFX Input plugin: https://www.mobileread.com/forums/showthread.php?t=291290

**Comment by ElleKayEm** [comment]
That's all the help I can give without more info on the problem.
